src/cc/common/StrUtils.cc

```diff
@@ -6,7 +6,7 @@
 namespace KFS {
 
 void
-StrUtils::str_printf_helper(string* dest, const char* format,
+StrUtils::printf_helper(string* dest, const char* format,
```

Comment:
Based on the recent e-mail exchange: should "_" characters be removed? also, should the function names follow camel case?